sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -164,7 +164,7 @@ public void forceFlush() {
 
     private static final BoundLongCounter droppedSpans;
 
-    private final Timer timer = new Timer(TIMER_THREAD_NAME);
+    private final Timer timer = new Timer(TIMER_THREAD_NAME, /* isDaemon= */ true);
```

Comment:
ugh. I'd like to get rid of the usage of Timer completely and replace it with a ScheduledExecutor